src/admin-views/settings/getting-started.php

```diff
@@ -14,7 +14,7 @@
 	: __( 'Thank you for using Event Tickets! We recommend looking through the settings below so that you can fine tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' );
 
 ?>
-<div class="event-tickets__admin-banner">
+<div class="event-tickets__admin-banner notice is-dismissible">
```

Comment:
And here remove the div wrapper. Then you can apply the particular padding and styles to match the design to this class `.tribe-notice-event-tickets-settings-getting-started`